server/controllers/siteMetadataController/index.js

```diff
@@ -3,11 +3,11 @@ import SiteMetadataModel from '../../models/SiteMetadataModel'
 const SiteMetadataController = {
```

Comment:
should this file be nested in an `api` directory so it's clear which controllers are api controllers and which aren't? For example, `controllers/api/siteMetadataController/index.js`

Reply:
[e8298f6](https://github.com/AMP-SCZ/dpdash/pull/440/commits/e8298f66e6af2e7b832152aab4bfbe381f3231a6)